nimbus/host-path/nimbus-deployment.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nimbus
+  namespace: nimbus
+  labels:
+    app: nimbus
+spec:
+  selector:
+    matchLabels:
+      app: nimbus
+  replicas: 1
+  strategy: 
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: nimbus
+    spec:
+      initContainers:
+      - name: init-wait
+        image: alpine:3.13.2
+        command: ['sh', '-c', 'echo Wait for 780 seconds for extra slashing protection! && sleep 780']
+      containers:
+      - name: nimbus
+        image: index.docker.io/statusim/nimbus-eth2:amd64-v1.2.1
+        args:
+        - --network=prater
+        - --non-interactive
+        - --data-dir=/data/nimbus
+        - --validators-dir=/data/nimbus/validators
+        - --secrets-dir=/data/nimbus/secrets
+        - --web3-url=<goerli eth1 node>
+        - --graffiti="eth2 x k8s"
+        - --log-level=INFO
```

Comment:
As mentioned in the other comment: We can remove this line since the default log level is INFO.